src/main/scala/no/ndla/listingapi/model/domain/search/SearchableCover.scala

```diff
@@ -32,5 +32,6 @@ case class SearchableCover(
   supportedLanguages: Seq[String],
   updatedBy: String,
   update: Date,
-  theme: String
+  theme: String,
+  oldNodeId: Long
```

Comment:
Det var kun fordi den dukka opp mens jeg skulle finne hvor den andre buggen var henne :P Dette burde i teorien ha vært to bug fix PRer  :see_no_evil: